Solutions/DNS Essentials/Analytic Rules/MultipleErrorsReportedForSameDNSQueryStaticThresholdBased.yaml

```diff
@@ -1,7 +1,7 @@
 id: 5b8344eb-fa28-4ac3-bcff-bc19d5d63089
 name: Detect DNSQueries reporting multiple errors from different client sources - Static threshold based (ASIM DNS Solution)
 description: |
-  'This rule creates an alert when multiple client sources reports errors for the same DNSQuery.\n\nThe rule utilize [ASIM](https://aka.ms/AboutASIM) normalization, and is applied to any source which supports the ASIM DNS schema'
+  'This rule creates an alert when multiple client sources report errors for the same DNSQuery. This rule monitors DNS traffic to detect  possible similar C2 communication. \n\nIt utilizes [ASIM](https://aka.ms/AboutASIM) normalization and is applied to any source which supports ASIM DNS schema'
```

Comment:
1. Single space between "detect" and "possible".

1. The two sentenced in the text seem repetative.

Reply:
this is done.